test/MusicStore.Test/AnnouncementComponentTest.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Mvc;
+using Microsoft.Data.Entity;
+using Microsoft.Framework.Caching.Memory;
+using Microsoft.Framework.Caching.Memory.Infrastructure;
+using Microsoft.Framework.DependencyInjection;
+using MusicStore.Models;
+using Xunit;
+
+namespace MusicStore.Components
+{
+    public class AnnouncementComponentTest
+    {
+        private readonly IServiceProvider _serviceProvider;
+
+        public AnnouncementComponentTest()
+        {
+            var services = new ServiceCollection();
+
+            services.AddEntityFramework()
+                      .AddInMemoryStore()
+                      .AddDbContext<MusicStoreContext>();
+
+            _serviceProvider = services.BuildServiceProvider();
+        }
+
+        [Fact]
+        public async Task AnnouncementComponent_Returns_LatestAlbum()
+        {
+            // Arrange
+            var today = DateTime.Parse("10/30/2012");
```

Comment:
You can just construct a DateTime (or DateTimeOffset, really) instead of parsing. Parsing is affected by culture, which we need to avoid.